CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Extrahorizon CLI changelog
 
+### vx.x.x
+* Added a validation check for `id` properties in arrays in data schemas, this will now be reported as an error if present.
```

Comment:
```suggestion
* Added a validation check for `id` properties in objects within arrays in data schemas, this will now be reported as an error if present.
```